src/Summoner/Tui.hs

```diff
@@ -156,7 +155,8 @@ draw :: Form SummonKit e SummonForm -> [Widget SummonForm]
 draw f =
     [ C.vCenter $ vBox
         [ C.hCenter form
-        , validationErrors $ invalidFields f
+      <+> W.hLimitPercent 25 (C.hCenter tree)
+        , validationErrors (invalidFields f)
```

Comment:
That's nice! I like new arrangement very much. Also it was unexpected that it works with just putting text there.